core/src/main/java/com/nerds/gamejam/util/CachingTextureLoader.java

```diff
@@ -0,0 +1,58 @@
+package com.nerds.gamejam.util;
+
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.Texture.TextureWrap;
+import com.badlogic.gdx.graphics.g2d.Animation;
+import com.badlogic.gdx.graphics.g2d.TextureRegion;
+import com.nerds.gamejam.ecs.component.AnimationComponent;
+import com.nerds.gamejam.ecs.component.TextureReferenceComponent.TextureReference;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+public class CachingTextureLoader implements TextureLoader {
```

Comment:
This class need to dispose of the textures so prolly needs a dispose() method, either bootstrap system or game jam itself can then kill it

Reply:
GameJam now creates + disposes of the texture loader